### PR TITLE
TEST - IGNORE

### PR DESCRIPTION
**Pull request by dcasavant**

### CODE REVIEW

File: charts/pega/values-cdh.yaml [hunk 1]
```diff
@@ -176,10 +176,10 @@ global:
           requests:
             storage: 5Gi
 
-    - name: "bix"
+    - name: "ADM"
```
Comment (taz-pega-work) [inline charts/pega/values-cdh.yaml:179]: please spell out what ADM stands for and update the comments here as well->
> Create a background tier for BIX processing.  This tier uses
> the BIX node type and will not be exposed to the load balancer.

File: charts/pega/values-cdh.yaml [hunk 2]
```diff
@@ -194,7 +194,27 @@ global:
         type: RollingUpdate
 
       hpa:
-        enabled: true
+        enabled: false
+
+    - name: "RTDG"
+      # Create a background tier for BIX processing.  This tier uses
```
Comment (taz-pega-work) [inline charts/pega/values-cdh.yaml:200]: please spell out what RTDG stands for and replace BIX in the comments here as well->
> Create a background tier for BIX processing.  This tier uses
> the BIX node type and will not be exposed to the load balancer

**Review by taz-pega-work — COMMENTED** [review]
minor comments that will be helpful to users inspecting these files

File: charts/pega/values-cdh.yaml (hunk 1 above)
Comment (s4rd1nh4) [reply charts/pega/values-cdh.yaml:179]: + standardize layer names to lowercase

Comment (s4rd1nh4) [inline charts/pega/values-cdh.yaml:179]: standardize layer names to lowercase

File: charts/pega/values-cdh.yaml [hunk 3]
```diff
@@ -194,7 +194,27 @@ global:
         type: RollingUpdate
 
       hpa:
-        enabled: true
+        enabled: false
+
+    - name: "RTDG"
```
Comment (s4rd1nh4) [inline charts/pega/values-cdh.yaml:199]: standardize layer names to lowercase

**Pull request closed without merging**